### PR TITLE
Switch to Coveralls to collect code coverage reports

### PR DESCRIPTION
**Pull request by domszyn**

### CODE REVIEW

**Comment by domszyn** [comment]
Can't get it to check out in CircleCI :(

**Comment by noseglid** [comment]
That's so weird. This is a public repository. No authorization required at all

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,13 +2,19 @@ version: 2.1
...
     docker:
-      - image: circleci/golang:1.12
+      - image: circleci/golang:1.15
         environment:
           GO111MODULE: "on"
-    working_directory: /go/src/github.com/{{ORG_NAME}}/{{REPO_NAME}}
     steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "54:0a:dd:d8:d1:ac:94:69:18:61:80:9f:90:28:b7:86"
```
Comment (noseglid) [inline .circleci/config.yml:11]:
```suggestion
```

Comment (noseglid) [reply .circleci/config.yml:11]: Don't need these.

**Review by noseglid — APPROVED** [review]

**Review by noseglid — APPROVED** [review]

**Pull request merged**